test/utilities/apexUtils.ts

```diff
@@ -18,26 +19,24 @@ export async function createApexClass(
   const workbench = await getWorkbench();
 
   // Using the Command palette, run SFDX: Create Apex Class to create the main class
-  const inputBox = await executeQuickPick('SFDX: Create Apex Class', Duration.seconds(1));
+  const inputBox = await executeQuickPick('SFDX: Create Apex Class', Duration.seconds(3)); // have to increase it otherwise it keeps failing on macos
 
   // Set the name of the new Apex Class
   await inputBox.setText(name);
-  await pause(Duration.seconds(1));
-  await browser.keys(['Enter']);
+  await inputBox.confirm();
   await pause(Duration.seconds(1));
 
   // Select the default directory (press Enter/Return).
-  await browser.keys(['Enter']);
-  await pause(Duration.seconds(1));
-
+  await inputBox.confirm();
+  await pause(Duration.seconds(2)); // have to hard pause for text editor to be ready
   // Modify class content
   const textEditor = await getTextEditor(workbench, name + '.cls');
   await textEditor.setText(classText);
   await textEditor.save();
   if (breakpoint) {
     await textEditor.toggleBreakpoint(breakpoint);
   }
-  await pause(Duration.seconds(1));
+  await pause(Duration.seconds(2));
```

Comment:
What is the pause for?

Reply:
The function `createApexClass` can be called multiple times in a row in its consumers such as `createApexClassWithTest` and `createApexClassWithBugs`. The pause here is to give enough time between saving the text and calling the next `SFDX: Create Apex Class`. I noticed that pauses with short time may cause failure in popping up the command palette. Increasing the pause time could help reduce flappers like that.